targets/audio

```diff
@@ -67,30 +71,35 @@ build_cras() {
     local pkgconfigpath=''
     local archgccflags=''
     if [ -n "$cras_arch" ]; then
-        pkgsuffix=":$cras_arch"
         pkgdepextra='gcc-multilib'
         archextrapath="/$cras_arch-linux-gnu"
         pkgconfigpath="/usr/lib$archextrapath/pkgconfig"
         archgccflags='-m32'
 
         # Add foreign architecture, if necessary
-        if ! dpkg --print-foreign-architectures | grep -q "^$cras_arch$"; then
-            echo "Adding foreign architecture $cras_arch to dpkg..." 1>&2
-            dpkg --add-architecture "$cras_arch"
-            apt-get update || true
-        fi
+        if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+            pkgsuffix=":$cras_arch"
+            if ! dpkg --print-foreign-architectures | grep -q "^$cras_arch$"; then
+                echo "Adding foreign architecture $cras_arch to dpkg..." 1>&2
+                dpkg --add-architecture "$cras_arch"
+                apt-get update || true
+            fi
+	fi
     fi
 
     # Install CRAS dependencies
     install --minimal alsa-utils \
-        libasound2$pkgsuffix libspeexdsp1$pkgsuffix
+        gentoo=media-libs/alsa-lib,libasound2$pkgsuffix gentoo=speex,libspeexdsp1$pkgsuffix
 
-    install --minimal --asdeps gcc $pkgdepextra libc6-dev$pkgsuffix \
-        pkg-config libspeexdsp-dev$pkgsuffix
+    # Gentoo already has these deps installed
+    if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+        install --minimal --asdeps gcc $pkgdepextra libc6-dev$pkgsuffix \
+            pkg-config libspeexdsp-dev$pkgsuffix
+    fi
 
     # precise does not allow libasound2-dev and libasound2-dev:i386 to be
     # installed simultaneously
-    if release -le precise && [ -n "$cras_arch" ]; then
+    if release -le precise && [ -n "$cras_arch" ] && [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
```

Comment:
`release -le precise` will return false if the distro isn't ubuntu/debian, so the extra condition is unnecessary